docs/BackendDiagram/Architecture.puml

```diff
@@ -0,0 +1,21 @@
+@startuml
```

Comment:
![image](https://user-images.githubusercontent.com/57187325/97543213-e1791e00-1a02-11eb-946f-a45674949670.png)

Not too sure if a class diagram is the correct format for the architecture of the application. I would recommend making of the diagram using Powerpoint!

The use of arrows is rather unclear here as to its purpose; I am not sure what the arrowhead means. Perhaps some annotation towards the arrow can be helpful in explaining the diagram better!